jsonnet/kube-prometheus/grafana/grafana.libsonnet

```diff
@@ -0,0 +1,85 @@
+local defaults = {
```

Comment:
any reason why we wouldn't just refactor kubernetes-grafana in the same way?

Reply:
I want to do this. But this PR as is allows me to parallelize work on kube-prometheus and kubernetes-grafana.

Reply:
Sounds very reasonable. Nice work!